wordpress-mobile/WordPress-iOS
# Fix incorrect media library count

The total WordPress Media Library images count displayed in media picker includes all media, whereas it should be just number of _images_.

See https://github.com/wordpress-mobile/WordPressKit-iOS/pull/620 for the cause of this issue.

## Test Instructions

1. Choose a site whose media library have images and documents.
2. Launch the app and go to the site
3. Tap the site icon on the My Site tab
4. Tap "Choose Image From My Device" action
5. Verify if the number displayed under "WordPress Media" is the total images count.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
What's described in the Test Instructions section

3. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:


UI Changes testing checklist: N/A
